### DISCUSSION TITLE
Builds stuck in Queue and don't start

### DISCUSSION THREAD

**Discussion by zd819** [codemagic-ci-cd/codemagic-docs#2725] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Trigger a build through the Codemagic website or by pushing to GitHub.
The build will stay in the queued status.

### Expected results

The build machine gets allocated and starts running the build pipeline.

### Actual results

The build status remains in queued for over an hour, and the build does not start.

**Comment by pstromberg98** [comment]
I am seeing this as well...again.

**Comment by den0206** [comment]
Same here, yesterday I was able to resolve and build with the following problem.
Seems to be a recurring [issue](https://github.com/orgs/codemagic-ci-cd/discussions/2723).

**Comment by mikhail-tokarev** [comment]
duplicate https://github.com/orgs/codemagic-ci-cd/discussions/2723
we will update the original post once we have more information
